#!/usr/bin/env python
PACKAGE = "scriba_vision"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()


gen.add("min_threshold",    int_t,    0, "min threshold", 2,  0, 15)
gen.add("ksize_height",    int_t,    0, "gaussian blur kernel height", 11,  0, 50)
gen.add("ksize_width",    int_t,    0, "gaussian blur kernel width", 11,  0, 50)


exit(gen.generate(PACKAGE, "scriba_vision", "ScribaVision"))
